dlrover/python/elastic_agent/monitor/training.py

```diff
@@ -119,9 +119,16 @@ def report_resource_with_step(self):
         except Exception as e:
             logger.warning(e)
 
-    def _periodically_report_step(self):
-        if self._group_rank != 0:
-            return
+    def send_heartbeat(self):
+        try:
+            ts = int(time.time())
```

Comment:
why should we use a int timestamp here? And `now = time.time()` is not converted to a int.

Reply:
Thanks for explaining, LGTM!